packages/sysweb3-keyring/src/transactions/syscoin.ts

```diff
@@ -623,9 +668,9 @@ export const SyscoinTransactions = (): ISyscoinTransactions => {
   ): Promise<ITxid> => {
     const { _hd, _main } = getSigners();
 
-    const { fee, assetGuid, assetWhiteList } = temporaryTransaction;
+    const { fee, assetGuid } = temporaryTransaction;
```

Comment:
We might have assetWhitelist in some scenarios as an optional parameter that if existent should be sent through txOptions